app/assets/javascripts/discourse/app/components/bulk-select-topics-dropdown.gjs

```diff
@@ -35,12 +36,23 @@ export function addBulkDropdownButton(opts) {
 }
 
 export default class BulkSelectTopicsDropdown extends Component {
+  @service router;
   @service modal;
   @service currentUser;
   @service siteSettings;
 
   get buttons() {
     let options = [
+      {
+        id: "dismiss",
+        icon: "check",
+        name: "Dismiss",
+        visible: ({ router }) => {
+          return ["discovery.unread", "discovery.new"].includes(
```

Comment:
Does the same modal work for both `unread` and `new`? I see a `dismiss-new.gjs` file, I suspect for new, we will need to use that modal.